### PR TITLE
feat(dedicated.nutanix): update licence tile for nutanix pack v2

### PR DESCRIPTION
**Pull request by darsene**
ref: MANAGER-13761

| Question         | Answer
| ---------------- | ---
| Branch?          | `feat/nutanix-package-v2`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-13761
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~Breaking change is mentioned in relevant commits~

## Description

Update the license tile for Nutanix pack v2 offer (with a restricted number of features) and ensure compatibility with legacy pack offer

### CODE REVIEW

File: packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-legacy-features/controller.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+export default class NutanixLicenceTileLegacyFeaturesCtrl {
+  /* @ngInject */
+  constructor(atInternet) {
+    this.atInternet = atInternet;
+  }
+
+  $onInit() {
+    this.defaultLicenseLength = 12;
```
Comment (ghyenne) [inline packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-legacy-features/controller.js:8]: Maybe move 12 in constant file.

File: packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-legacy-features/template.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+>
+    <oui-tile-term data-ng-bind=":: license.getFormatedName()"></oui-tile-term>
+    <oui-tile-description>
+        <span
+            class="oui-badge oui-badge_success"
+            data-ng-if="license.isEnabled() && license.name !== 'lifeCycleManager'"
+            data-translate="nutanix_cluster_licence_included"
+        ></span>
+        <span
+            data-ng-if="!license.isEnabled() && license.name !== 'files' && license.name !== 'objects'"
```
Comment (ghyenne) [inline packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-legacy-features/template.html:12]: I don't know if it's possible to put less conditional logic in html file. Maybe use a function in controller.

File: packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-legacy-features/controller.js (hunk 1 above)
Comment (darsene) [reply packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-legacy-features/controller.js:8]: It's legacy code and only use in that template so I let as is

**Review by ghyenne — APPROVED** [review]

File: packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-legacy-features/template.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
+<oui-tile-definition
+    data-ng-repeat="license in $ctrl.licenses.getFeatures() | limitTo: ($ctrl.expand ? $ctrl.licenses.getFeatures().length : $ctrl.defaultLicenseLength) track by license.name"
+>
+    <oui-tile-term data-ng-bind=":: license.getFormatedName()"></oui-tile-term>
```
Comment (dectotam) [inline packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-legacy-features/template.html:4]:
```suggestion
    <oui-tile-term data-ng-bind=":: license.getFormattedName()"></oui-tile-term>
```

File: packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-features/template.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<oui-tile-definition
+    data-ng-repeat="license in $ctrl.licenses.getFeatures() track by license.name"
+>
+    <oui-tile-term
+        data-translate="{{:: 'nutanix_cluster_cloud_licence_' + license.getFormatedName() }}"
```
Comment (dectotam) [inline packages/manager/modules/nutanix/src/dashboard/component/licence-tile/licence-tile-features/template.html:5]: Wrong spelling here

```suggestion
        data-translate="{{:: 'nutanix_cluster_cloud_licence_' + license.getFormattedName() }}"
```

File: packages/manager/modules/nutanix/src/dashboard/component/licence-tile/translations/Messages_de_DE.json [hunk 5]
```diff
@@ -6,4 +6,4 @@
   "nutanix_cluster_licence_show_less": "Weniger Optionen anzeigen",
   "nutanix_cluster_licence_show_files_objects_size": "Inklusive (maximal {{size}} TB",
   "nutanix_cluster_licence_show_LCM": "Inklusive (nur Software Upgrade)"
-}
\ No newline at end of file
+}
```
Comment (dectotam) [inline packages/manager/modules/nutanix/src/dashboard/component/licence-tile/translations/Messages_de_DE.json:9]: maybe roolback this space change so no translation file other that the french one is modified ?

**Review by dectotam — APPROVED** [review]

**Pull request merged**